examples/AdvancedConsumer/Program.cs

```diff
@@ -189,7 +189,7 @@ public static void Run_Consume(string brokerList, List<string> topics)
                 consumer.OnError += (_, error)
                     => Console.WriteLine($"Error: {error}");
 
-                consumer.OnOffsetCommit += (_, commit) =>
+                consumer.OnOffsetsCommitted += (_, commit) =>
```

Comment:
comment above was just in the wrong spot: In this example we are manually committing offsets and still registered for the OnOffsetsCommitted event which is no longer relevant.